app/src/main/res/layout/drawer_header.xml

```diff
@@ -3,14 +3,15 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/navigation_drawer_header"
-    android:layout_width="wrap_content"
+    android:layout_width="match_parent"
     android:layout_height="@dimen/nav_drawer_header_height">
 
-    <ImageView
+    <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:scaleType="centerCrop"
-        android:src="@drawable/navigation_drawer_header_background" />
```

Comment:
You have removed the image reference at this point, but the PR replaces the image file with a new one. Is `@drawable/navigation_drawer_header_background` used anywhere else?